tasks/redhat.yml

```diff
@@ -3,65 +3,72 @@
   yum:
     name: epel-release
     state: present
-  become: true
   register: result
   until: result is successful
 
 - name: redhat | Installing Pre-Reqs
   yum:
     name: "{{ postgresql_redhat_pre_reqs }}"
     state: present
-  become: true
   register: result
   until: result is successful
 
 - name: redhat | Installing Development Tools
   yum:
     name: "@Development tools"
     state: present
-  become: true
   register: result
   until: result is successful
 
 - name: redhat | Installing Postgresql Python Modules
   pip:
     name: ["psycopg2"]
     state: present
-  become: true
   register: result
   until: result is successful
 
 - name: redhat | Adding Postgresql Repo
   yum:
     name: https://download.postgresql.org/pub/repos/yum/reporpms/EL-7-x86_64/pgdg-redhat-repo-latest.noarch.rpm
     state: present
-  become: true
   register: result
   until: result is successful
 
+- name: redhat | Create version for paths
+  set_fact:
+    _postgresql_version: "{{ postgresql_version | replace('.', '') }}"
+
+- name: redhat | Facts for paths/commands
```

Comment:
Unified this a little. Otherwise you have to rebuild paths again and again. Maybe this should also go into your facts setup?

Reply:
Unification for the version and some paths.

Reply:
I'd probably add the version specs to `set_facts` as you mentioned. They are going to be different for each distribution anyways and `10/11` change it even more from what we have seen.